build.sh

```diff
@@ -1,48 +1,91 @@
-DIST=$PWD/"dist"
+#!/bin/bash
+ffmpeg_proj_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-if [ ! -d $DIST ]; then
-    mkdir $DIST
-    echo "CREATED" $DIST
+DIST=${ffmpeg_proj_dir}/"dist"
```

Comment:
Wasn't the other way to set DIST simpler?

Reply:
The other way is safe only if the script is run in the same repo (e.g. `./build.sh`).  If someone chose to run this script like so:

```bash
cd ~/git-src
FFmpeg/build.sh
```

even if the rest fails, this will create a `dist` dir in the location the script was run (which is `~/git-src` in this example). The stuff before it checks what is the directory `build.sh` is in, and makes changes there.